am-lib/src/main/java/uk/gov/hmcts/reform/amlib/utils/Permissions.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.amlib.utils;
+
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.exceptions.UnsupportedPermissionsException;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public final class Permissions {
+
+    private Permissions() {
+
+    }
+
+    public static int sumOf(Set<Permission> perms) {
+        int sum = 0;
+
+        for (Permission permission : perms) {
+            sum += permission.getValue();
+        }
+
+        return sum;
+    }
+
+    /**
+     * Builds a list of permissions based on integer value. HIDE is removed from values above 0, as HIDE only
```

Comment:
Same here, remove HIDE from Javadoc since it got removed from code.

Reply:
👍